Libraries/LibWeb/TrustedTypes/TrustedTypePolicyFactory.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2025, Miguel Sacristán Izcue <miguel_tete17@hotmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibJS/Forward.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/Bindings/TrustedTypePolicyFactoryPrototype.h>
+
+namespace Web::TrustedTypes {
+
+class TrustedTypePolicyFactory final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(TrustedTypePolicyFactory, Bindings::PlatformObject);
+    GC_DECLARE_ALLOCATOR(TrustedTypePolicyFactory);
+
+public:
+    [[nodiscard]] static GC::Ref<TrustedTypePolicyFactory> create(JS::Realm&);
+
+    virtual ~TrustedTypePolicyFactory() override { }
+
+    Optional<String> get_attribute_type(String&, String&, Optional<String>, Optional<String>);
```

Comment:
It would probably be good to name the arguments here, since it isn't obvious what each string is supposed to represent at a glance.

Reply:
To be clear, I think it would be better like this: 
```cpp
Optional<String> get_attribute_type(String& tag_name, String& attribute, Optional<String> element_ns, Optional<String> attr_ns);`
```

Also `tag_name` can be `const&` here.